lib/restaurant.rb

```diff
@@ -0,0 +1,40 @@
+require 'twilio-ruby'
+require_relative 'twilio_credentials'
+
+class Restaurant
+
+  WAIT_TIME = 3600 # in seconds
+
+  def initialize
+    @account_sid = TWILIO_ACCOUNT_SID
+    @auth_token = TWILIO_AUTH_TOKEN
+    @from = FROM
+  end
+
+  def place(order, number = nil)
+    @order = order
+    deadline = wait.strftime('%H:%M')
+    message = "Thank you! Your order was placed and will be delivered before #{deadline}"
+    return message if number == nil?
+    send_sms(message, number)
+  end
+
+  private
+
+  def send_sms(message, number)    
+    # set up a client to talk to the Twilio REST API
+    @client = Twilio::REST::Client.new @account_sid, @auth_token
```

Comment:
To test this without sending an SMS every time, you'll need to inject the client (so that you can then inject a double in your tests).